Source/WebCore/dom/Document.h

```diff
@@ -325,7 +325,11 @@ enum class DocumentCompatibilityMode : uint8_t {
     LimitedQuirksMode = 1 << 2
 };
 
-enum DimensionsCheck { WidthDimensionsCheck = 1 << 0, HeightDimensionsCheck = 1 << 1, AllDimensionsCheck = 1 << 2 };
+enum class DimensionsCheck : uint8_t {
+    Width = 1 << 0,
+    Height = 1 << 1,
+    All = 1 << 2,
```

Comment:
We should probably add a FIXME comment here, `All` probably meant to be `Width | Height`.

Reply:
Yeah, will add a FIXME and follow up.